Competition2017/src/competition/subsystems/climbing/RopeAlignerSubsystem.java

```diff
@@ -0,0 +1,39 @@
+package competition.subsystems.climbing;
+
+import org.apache.log4j.Logger;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import xbot.common.command.BaseSubsystem;
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+@Singleton
+public class RopeAlignerSubsystem extends BaseSubsystem {
+    private static Logger log = Logger.getLogger(RopeAlignerSubsystem.class);
+    
+    public final XCANTalon intakeMotor;
```

Comment:
It seems the opposite to me. We need to protect ourselves from this because we have a bunch of new programmers who could make the mistake of accessing this field, and have demonstrated that we don't catch all of these issues in review.

Importantly, the work to add these classes is minimal: it's so easy that writing it yourself is less work than copy-pasting. The most recent implementation of this sort is [here](https://github.com/Team488/TeamXbot2017/pull/20/files#diff-58e80124d733054e49f238191cb87871) -- one class with the default constructor (Eclipse does that for you) and a method per property.